enterprise/users/src/main/java/io/crate/auth/user/StatementPrivilegeValidator.java

```diff
@@ -139,6 +140,12 @@ public Void visitAlterUserStatement(AlterUserAnalyzedStatement analysis, User us
             return null;
         }
 
+        @Override
+        public Void visitDecommissionNode(AnalyzedDecommissionNodeStatement decommissionNode, User user) {
+            throwUnauthorized(user.name());
```

Comment:
I wonder if we should make `throwUnauthorized` the default for any statement that is not explicitly handled?

Reply:
+1 wonder why we've not did it like that